html/themes/custom/common_design_subtheme/templates/form/node-edit-form--training.html.twig

```diff
@@ -16,6 +16,7 @@
  */
 #}
 {{ attach_library('common_design_subtheme/rw-form') }}
+{{ attach_library('common_design_subtheme/rw-user-information') }}
```

Comment:
Now that there is a separate component for the user information, I wonder if it would not better to create an override of `reliefweb_entities/templates/reliefweb-entities-form-user-information.html.twig` in the subtheme and attack this library there. What do you think?